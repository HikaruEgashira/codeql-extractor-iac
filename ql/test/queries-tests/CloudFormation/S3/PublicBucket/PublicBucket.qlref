security/CloudFormation/S3/PublicBucket.ql