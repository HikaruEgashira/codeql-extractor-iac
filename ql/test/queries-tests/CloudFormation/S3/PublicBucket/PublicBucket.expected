| s3.yml:4:5:12:2 | CloudFormation Resource | Public S3 Bucket resource |
